### PR TITLE
Update Makefile to push releases to Stacksmith

### PR DESCRIPTION
**Pull request by prydonius**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,6 +34,14 @@ push: build
 			if [ "$$revision" != "DEV" ]; then \
 				gcloud docker -- push gcr.io/$(GCLOUD_PROJECT)/$$distro:$$release ; \
 				gcloud docker -- push gcr.io/$(GCLOUD_PROJECT)/$$distro-buildpack:$$release ; \
+				if [ -n "$(STACKSMITH_API_KEY)" ]; then \
+					export jessie_version=8.6 ; \
```
Comment (prydonius) [inline Makefile]: This is a mapping we would need to create for each `release`.

**Comment by prydonius** [comment]
I'm wondering if we want to make this Makefile less generic, and either just add a specific task for `jessie`/`minideb` or if we should move logic out to shell scripts. It's getting difficult to maintain like this.

File: Makefile (hunk 1 above)
Comment (sameersbn) [inline Makefile]: Will make the required changes in the circle env and test this out. thanks for this PR :+1:

**Comment by sameersbn** [comment]
> It's getting difficult to maintain like this.

Yep. Will move it out. We should be doing bulk of this stuff in scripts.

**Pull request merged**